default-plugins/status-bar/src/main.rs

```diff
@@ -41,6 +32,153 @@ impl Display for LinePart {
+    pub unselected_suffix_separator: Style,
+    // disabled mode
+    pub disabled_prefix_separator: Style,
+    pub disabled_styled_text: Style,
+    pub disabled_suffix_separator: Style,
+    // selected single letter
+    pub selected_single_letter_prefix_separator: Style,
+    pub selected_single_letter_char_shortcut: Style,
+    pub selected_single_letter_suffix_separator: Style,
+    // unselected single letter
+    pub unselected_single_letter_prefix_separator: Style,
+    pub unselected_single_letter_char_shortcut: Style,
+    pub unselected_single_letter_suffix_separator: Style,
+    // superkey
+    pub superkey_prefix: Style,
+    pub superkey_suffix_separator: Style,
+}
+
+impl ColoredElements {
+    #[allow(clippy::too_many_arguments)]
+    pub fn new(
+        selected_prefix_separator: Style,
+        selected_char_left_separator: Style,
+        selected_char_shortcut: Style,
+        selected_char_right_separator: Style,
+        selected_styled_text: Style,
+        selected_suffix_separator: Style,
+        unselected_prefix_separator: Style,
+        unselected_char_left_separator: Style,
+        unselected_char_shortcut: Style,
+        unselected_char_right_separator: Style,
+        unselected_styled_text: Style,
+        unselected_suffix_separator: Style,
+        disabled_prefix_separator: Style,
+        disabled_styled_text: Style,
+        disabled_suffix_separator: Style,
+        selected_single_letter_prefix_separator: Style,
+        selected_single_letter_char_shortcut: Style,
+        selected_single_letter_suffix_separator: Style,
+        unselected_single_letter_prefix_separator: Style,
+        unselected_single_letter_char_shortcut: Style,
+        unselected_single_letter_suffix_separator: Style,
+        superkey_prefix: Style,
+        superkey_suffix_separator: Style,
+    ) -> Self {
+        Self {
+            selected_prefix_separator,
+            selected_char_left_separator,
+            selected_char_shortcut,
+            selected_char_right_separator,
+            selected_styled_text,
+            selected_suffix_separator,
+            unselected_prefix_separator,
+            unselected_char_left_separator,
+            unselected_char_shortcut,
+            unselected_char_right_separator,
+            unselected_styled_text,
+            unselected_suffix_separator,
+            disabled_prefix_separator,
+            disabled_styled_text,
+            disabled_suffix_separator,
+            selected_single_letter_prefix_separator,
+            selected_single_letter_char_shortcut,
+            selected_single_letter_suffix_separator,
+            unselected_single_letter_prefix_separator,
+            unselected_single_letter_char_shortcut,
+            unselected_single_letter_suffix_separator,
+            superkey_prefix,
+            superkey_suffix_separator,
+        }
+    }
+}
+
+// I really hate this, but I can't come up with a good solution for this,
```

Comment:
So the default and Xresources are actually themed differently? As opposed to just having a different palette?

Reply:
That was only needed, because I wanted to keep the original default coloring, and as for the Xresources I wanted to make it slightly different, because if mapped as default it looked horrific and vice versa. Also I think that could be moved somewhere else or something similar to be done to color individual elements of Zellij into whatever you want(as in one of the discussions we started on discord), not sure if this is a cool feature or not tho